exonum-java-binding-core/src/test/java/com/exonum/binding/storage/indices/CheckedMapProofMatcherTest.java

```diff
@@ -66,22 +66,21 @@ public void matchesValidProof() {
 
   @Test
   public void describeMismatchSafelyCorrectProof() {
-    HashCode key = HashCode.fromString("ab");
+    HashCode key = TEST_KEY;
     String expectedValue = null;  // No value
     CheckedMapProofMatcher matcher = CheckedMapProofMatcher.isValid(key, expectedValue);
 
-    byte[] actualValue = StandardSerializers.string().toBytes("value");
+    byte[] actualValue = StandardSerializers.string().toBytes(TEST_VALUE);
     MapEntry entry = new MapEntry(key.asBytes(), actualValue);
-    HashCode rootHash = HashCode.fromString("123456ef");
     CheckedMapProof proof = CheckedFlatMapProof.correct(
-        rootHash,
+        ROOT_HASH,
         Collections.singletonList(entry),
         Collections.emptyList());
 
     Description d = new StringDescription();
     matcher.describeMismatchSafely(proof, d);
 
-    assertThat(d.toString(), equalTo("was a valid proof, entries=[(ab -> value)], "
+    assertThat(d.toString(), equalTo("was a valid proof, entries=[(ab -> hello)], "
```

Comment:
I leaved this constants to aid readability of this message, so that you know where ab, hello and 12…ef come from.

Reply:
Ok. I'd prefer to use constants in test body and their string representations in `assertThat(...)`, but maybe it'd be too verbose.